src/fund/utils/subscribeToWindowMessage.ts

```diff
@@ -0,0 +1,85 @@
+import { DEFAULT_ONRAMP_URL } from '../constants';
+import type { JsonObject } from '../jsonTypes';
+
+export enum MessageCodes {
+  LaunchEmbedded = 'launch_embedded',
+  AppReady = 'app_ready',
+  AppParams = 'app_params',
+  SigninSuccess = 'signin_success',
+  Success = 'success',
+  Exit = 'exit',
```

Comment:
@rustam-cb does this message still get published? Seems like we have an "event" message that is an exit event. We probably shouldn't have two versions of exit.

Same goes for error and exit. Can you see if those are maybe legacy events that we don't need to support any more?

Reply:
`Error` is not used so removing it from here. 

`Exit` is still used.

Reply:
Isn't there an exit event?
```
export type ExitEvent = {
  eventName: 'exit';
  // biome-ignore lint/suspicious/noExplicitAny: <explanation>
  error?: any;
};
```
What's the difference?

Reply:
I'm a bit confused by these Events and MessageCode and which one we need to deprecate from those. Did you mean to remove `exit` from the MessageCode?

Reply:
Yeah I think so, can you take a quick look at the history of the old SDK and see why we have both a message code called exit and an event called exit? My impression was that the event was put it to replace the message code, and the message code was kept for backwards compatibility. But we should confirm this and just remove anything that's old.

Reply:
I could not find the reason why it was done as the event was just added without much context. I see that messageCode is only used in `embed` files, so I will remove it from the messageCode. 

Its still not fully clear to me what is going on there (not the flow but with all the structure like:

```
{
  eventName: 'something',
  data: {
     eventName: 'somethingELSE'
  }
}
```

But I think as I dig more into it I'll be able to better adjust it.